uiowa/uiowa
# [Commencement] Ceremonies page

Resolves #7649 

# How to test

```
ddev blt ds --site commencement.uiowa.edu
```
- https://commencement.uiowa.ddev.site/admin/structure/taxonomy/manage/session/overview
- Edit a session to set a default session
- https://commencement.uiowa.ddev.site/ceremonies
- Confirm that selected session in session drop down matches your selection.
- Set a different default session and ensure that it updates in the views exposed filter without needing to clear caches.
- Review these requirements to make sure the view page meets them: https://github.com/uiowa/uiowa/issues/7649#issue-2278252914
